lib/task_bunny/config.ex

```diff
@@ -152,6 +152,18 @@ defmodule TaskBunny.Config do
   end
 
   @doc """
+  Returns true if worker is disabled.
+  """
+  @spec disable_worker? :: boolean
+  def disable_worker? do
+    env = System.get_env("TASK_BUNNY_DISABLE_WORKER") || ""
+    ["1", "true", "yes"]
```

Comment:
What about
```
env
|> String.downcase
|> Kernel.in(["1", "true", "yes"])
```
or
```
"TASK_BUNNY_DISABLE_WORKER"
|> System.get_env
|> Kernel.||("")
|> String.downcase
|> Kernel.in(["1", "true", "yes"])
```

That would turn the triple downcase into a single downcase, plus saves the creation of the anonymous function.

You could even do:
```
@worker_disable_env_var "TASK_BUNNY_DISABLE_WORKER"
@worker_disable_truthy ["1", "true", "yes"]
```
Though you're only using it in one place.